libraries/AP_ServoRelayEvents/AP_ServoRelayEvents.cpp

```diff
@@ -106,10 +106,6 @@ void AP_ServoRelayEvents::update_events(void)
         return;
     }
 
-    if (channel > NUM_SERVO_CHANNELS || channel == 0) {
```

Comment:
This check could be reworked to be correct in the case of a repeat servo or a repeat relay, but the same sort of checks are performed again immediately afterward before outputting. It's redundant.